lib/ansible/modules/cloud/amazon/ecs_ecr.py

```diff
@@ -358,7 +371,8 @@ def main():
                    default='present'),
         force_set_policy=dict(required=False, type='bool', default=False),
         policy=dict(required=False, type='json'),
-        delete_policy=dict(required=False, type='bool')))
+        delete_policy=dict(required=False, type='bool'),
+        force_delete=dict(required=False, type='bool', default=False)))
 
     module = AnsibleModule(argument_spec=argument_spec,
```

Comment:
I know you didn't write the original module, but it would be good to switch this over to AnsibleAWSModule:

```suggestion
    module = AnsibleAWSModule(argument_spec=argument_spec,
```

Additionally:
- `from ansible.module_utils.aws.core import AnsibleAWSModule`
- Remove ec2_argument_spec (argument_spec only needs to include this module's extra options)
- remove the "HAS_BOTO3" check below: it's handled automatically for you
- Where the connections are initialized above (get_aws_connection_info, boto3_conn) can be simplified to:
```
self.ecr = module.client('ecr')
self.sts = module.client('sts')
```

Reply:
Will be treated in a dedicated PR

Reply:
See #62985